garage.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |s|
   s.add_dependency "hashie"
   s.add_dependency "sass-rails"
   s.add_dependency "coffee-rails"
-  s.add_dependency "http_accept_language", "~> 1.0.2"
+  s.add_dependency "http_accept_language", "~> 2.0.0"
```

Comment:
Please use `>=` instead of `~>` from this time because we should not prevent users from using uppper version. I think the problems with uppper version of dependecies should be able to be handled by users.